docker-app/qfieldcloud/notifs/tests/test_notifs.py

```diff
@@ -32,12 +32,12 @@ def setUpClass(cls):
         def noop():
             pass
 
-        runcrons.close_connection = noop
+        runcrons.close_old_connections = noop
 
     @classmethod
     def tearDownClass(cls):
         # restore above monkeypatch
-        runcrons.close_connection = cls._close_old_connections
+        runcrons.close_old_connections = cls._close_old_connections
```

Comment:
Test fix necessary because https://github.com/Tivix/django-cron/compare/v0.5.0...v0.6.0#diff-cc435718a61d956acbd90ddea2f08d79441e5df2f7fac06608e8c05a58e2fc50R64

Should only affect the test monkey patch.

Reply:
Thanks!